ratpack-core/src/main/java/ratpack/path/internal/DefaultPathBinding.java

```diff
@@ -66,18 +48,65 @@ public DefaultPathBinding(String binding, ImmutableMap<String, String> tokens, P
     }
   }
 
+  private static PathTokens mergeTokens(PathTokens thisTokens, PathTokens parentTokens) {
+    if (parentTokens.isEmpty()) {
+      return thisTokens;
+    } else if (thisTokens.isEmpty()) {
+      return parentTokens;
+    } else {
+      Map<String, String> merge = new LinkedHashMap<>(parentTokens);
+      merge.putAll(thisTokens);
+      return DefaultPathTokens.of(ImmutableMap.copyOf(merge));
+    }
+  }
+
+  @Override
+  public String getSpec() {
+    return description;
```

Comment:
Seems weird that the method is called spec, but the field is called description.